pandas/tests/arrays/integer/test_dtypes.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
```

Comment:
this overlaps a lot with test_casting.py can you just make one and/or make it clear what is in one vs the other.

Reply:
Moved casting tests into `test_dtypes.py`